### PR TITLE
Converted the class component to function component

### PR DESCRIPTION
**Pull request by myteax**

### CODE REVIEW

File: src/App2.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
+import React, { Component } from "react"
```
Comment (cujarrett) [inline src/App2.js:1]: @myteax - Did you mean to have this second App.js file?

**Comment by myteax** [comment]
App2.js is just a duplicate of the class component, App.js is the function
based component. I just kept App2.js for reference purposes. You can delete
it

File: src/App2.js (hunk 1 above)
Comment (myteax) [reply src/App2.js:1]: App2.js is the class component you did.  It can be deleted if you do not need it again

**Comment by cujarrett** [comment]
@myteax - I’ve been working on this branch locally and seem to find it doesn’t rotate pallets on random call, can you check it’s behavior on your local?

**Comment by myteax** [comment]
I will check it now

**Comment by myteax** [comment]
It is rotating the pallets on random call here, If i get you correctly, when you refresh the page, the background color does not change?

**Comment by cujarrett** [comment]
@myteax - Sorry for my description earlier. What I mean is while the order of colors does change on refresh or randomize button press, the color pallets do not rotate. It seems to stick with the original five colors. I think we need to look into the setPalet call where it sets it to current art colors.

**Comment by myteax** [comment]
I have fixed the palette rotating issue. Check and let me know if you have other concerns

**Comment by cujarrett** [comment]
@myteax - I really appriciate your help! I was working on solving the palette bug while you were, can you review #6  and we'll let that one check this box?

**Comment by myteax** [comment]
Approved.

**Pull request closed without merging**